Pipfile

```diff
@@ -17,6 +18,10 @@ google-cloud-storage = "*"
 pandas = "*"
 matplotlib = "*"
 slackclient = "*"
+pytest = "*"
```

Comment:
Does the pytest dependencies need to go into the dev-packages?

Reply:
Moved these now 👍